{
  "uid" : "48349d5cc15fdb28",
  "name" : "test_send_url",
  "fullName" : "test.test_messages_functionality#test_send_url",
  "historyId" : "4b0586f5f54c42dea3d6cdc7b39df69c",
  "time" : {
    "start" : 1712056717708,
    "stop" : 1712056717708,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "FileNotFoundError: File target.json not found",
  "statusTrace" : "request = <SubRequest 'config' for <Function test_get_user_ban>>\n\n    @pytest.fixture(scope=\"session\", autouse=True)\n    def config(request):\n        if \"GITHUB_ACTIONS\" in os.environ and os.environ[\"GITHUB_ACTIONS\"] == \"true\":\n            # Возвращаем данные из переменных окружения\n            return {\n                \"web\": {\n                    \"baseUrl\": os.getenv(\"BASE_URL\")\n                },\n                \"api\": {\n                    \"baseUrl\": os.getenv(\"API_BASE_URL\"),\n                    \"ssoUrl\": os.getenv(\"SSO_URL\"),\n                    \"X-Node-Id\": os.getenv(\"X_NODE_ID\"),\n                    \"Login_player\": os.getenv(\"LOGIN_PLAYER\"),\n                    \"Password_player\": os.getenv(\"PASSWORD_PLAYER\"),\n                    \"Login_admin\": os.getenv(\"LOGIN_ADMIN\"),\n                    \"Password_admin\": os.getenv(\"PASSWORD_ADMIN\")\n                }\n            }\n        else:\n            # Проверяем наличие файла target.json\n            config_file = request.config.getoption(\"--target\")\n            if not os.path.exists(config_file):\n>               raise FileNotFoundError(f\"File {config_file} not found\")\nE               FileNotFoundError: File target.json not found\n\nconftest.py:31: FileNotFoundError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "config",
    "time" : {
      "start" : 1712056717419,
      "stop" : 1712056717419,
      "duration" : 0
    },
    "status" : "broken",
    "statusMessage" : "FileNotFoundError: File target.json not found\n",
    "statusTrace" : "  File \"/usr/local/lib/python3.9/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.9/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/usr/local/lib/python3.9/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.9/site-packages/_pytest/fixtures.py\", line 913, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/usr/workspace/conftest.py\", line 31, in config\n    raise FileNotFoundError(f\"File {config_file} not found\")\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "stepsCount" : 0,
    "hasContent" : true
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "feature",
    "value" : "Sending URLs"
  }, {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "story",
    "value" : "Sending Allowed and Disallowed URLs"
  }, {
    "name" : "parentSuite",
    "value" : "test"
  }, {
    "name" : "suite",
    "value" : "test_messages_functionality"
  }, {
    "name" : "host",
    "value" : "43e0a1e0dd95"
  }, {
    "name" : "thread",
    "value" : "1-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test.test_messages_functionality"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 4,
        "unknown" : 0,
        "total" : 8
      },
      "items" : [ {
        "uid" : "292880dc8d71faba",
        "status" : "broken",
        "statusDetails" : "FileNotFoundError: [Errno 2] No such file or directory: '/usr/workspace/target.json'",
        "time" : {
          "start" : 1712056501130,
          "stop" : 1712056501130,
          "duration" : 0
        }
      }, {
        "uid" : "3ca6ee367fd55367",
        "status" : "broken",
        "statusDetails" : "FileNotFoundError: [Errno 2] No such file or directory: '/usr/workspace/target.json'",
        "time" : {
          "start" : 1712055502443,
          "stop" : 1712055502443,
          "duration" : 0
        }
      }, {
        "uid" : "da3ac1cf93b2c5c0",
        "status" : "broken",
        "statusDetails" : "FileNotFoundError: [Errno 2] No such file or directory: '/usr/workspace/target.json'",
        "time" : {
          "start" : 1712053563313,
          "stop" : 1712053563313,
          "duration" : 0
        }
      }, {
        "uid" : "1cfe2498374e1d6c",
        "status" : "passed",
        "time" : {
          "start" : 1712052153295,
          "stop" : 1712052154094,
          "duration" : 799
        }
      }, {
        "uid" : "625c0241b85f4d57",
        "status" : "passed",
        "time" : {
          "start" : 1712049661528,
          "stop" : 1712049662233,
          "duration" : 705
        }
      }, {
        "uid" : "604ecc363c9deacc",
        "status" : "passed",
        "time" : {
          "start" : 1712042044199,
          "stop" : 1712042045459,
          "duration" : 1260
        }
      }, {
        "uid" : "91e6232b5252b6a4",
        "status" : "passed",
        "time" : {
          "start" : 1711622003867,
          "stop" : 1711622004902,
          "duration" : 1035
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "48349d5cc15fdb28.json",
  "parameterValues" : [ ]
}